_posts/2016-01-16-swarm-scales-your-containter-for-free.markdown

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title:  "Swarm scales docker for free"
+date:   2015-12-14 10:08:27
+categories: devops
+img: /img/docker.png
+tags: devops, docker, swarm, gourmet
+summary: "Docker is an awesome tool to manage your container.
+Swarm helps you to scale your containers on more servers."
+priority: 0.6
+changefreq: yearly
+---
+
+Serverless, container, Aws lambda, [Gourmet](https://github.com/gianarb/gourmet) burs on this ideas,
+to improve my go knowledge and to work with the Docker API
+I am happy to share my idea and my tests with Swam an easy way to scale this type of application. 
+
+Gourmet exposes an HTTP api, route `/project`, method POST that wait a body similar to
```

Comment:
Better something like:

Gourmet exposes an HTTP API available at the `/project` endpoint that accept a JSON request body like: